spec/models/foi_attachment_spec.rb

```diff
@@ -67,6 +67,15 @@
       expect(main.body).to eq(orig_body)
     end
 
+    it 'can parse raw email and read attachment inside DB transaction' do
```

Comment:
No commits should break the build; should fix this up into the commit that fixes the failing spec.

Reply:
Okay. Think it warrants having this commit first as it shows the issue which we're fixing. I will change this commit to `xit` and change it back to `it` when fixed.